libraries/chain/include/eosio/chain/transaction.hpp

```diff
@@ -267,6 +267,8 @@ namespace eosio { namespace chain {
       prunable_data_type  prunable_data;
    };
 
+   using packed_transaction_v0_ptr = std::shared_ptr<const packed_transaction_v0>;
```

Comment:
Put this above `prunable_transaction_data`.

Reply:
moved